### PR TITLE
SDANSA-109 Support for multiple items on custom media fields

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

**Comment by petrjasek** [comment]
pls add some description how it works

**Comment by jerome-poisson** [comment]
Hi, I've tried the patch, it's working but I found a bug: if you drag and drop several files (tried with 3 files) at once, you see 3 panels to enter metadata as expected, but when saving only one file is visible.
If you want to add several files, you have to add them one by one.
Also it would be nice to activate multi-selection when uploading files, for now we can only select one at a time.

File: scripts/apps/authoring/authoring/directives/ItemAssociationDirective.js [hunk 1]
```diff
@@ -278,15 +293,31 @@ export function ItemAssociationDirective(superdesk, renditions, config, authorin
...
+
+                                function editNextFile() {
+                                    if (imagesWithIds.length > 0) {
+                                        var imageWithId = imagesWithIds.shift();
+
+                                        scope.edit(imageWithId.image, imageWithId.id, editNextFile);
+                                    }
+                                }
+
+                                _.forEach(images, (image) => {
```
Comment (petrjasek) [inline scripts/apps/authoring/authoring/directives/ItemAssociationDirective.js]: do we actually need user to edit all images right after uploading them?

Comment (ioanpocol) [reply scripts/apps/authoring/authoring/directives/ItemAssociationDirective.js]: This was the flow when you uploaded one image

Comment (mugurrus) [reply scripts/apps/authoring/authoring/directives/ItemAssociationDirective.js]: This was how the functionality was implemented. I guess we should ask if we want to change that, but I'd do that in another PR.

Comment (petrjasek) [reply scripts/apps/authoring/authoring/directives/ItemAssociationDirective.js]: I see

**Pull request merged**